desktop/app/main.js

```diff
@@ -27,6 +27,7 @@ const mainWindow = new Window('index', {
 
 mb.on('ready', () => {
   require('../../react-native/react/native/notifications').init()
```

Comment:
Hm, this is the very first JS file loaded by Electron, so we only have access to Babel's `import` in files that we require from this one.